.github/workflows/linux-32bit-build-and-test.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
         DEBIAN_FRONTEND: noninteractive
         IGNORES: "append-* debug_notice transparent_decompression-*
           transparent_decompress_chunk-* plan_skip_scan-12 pg_dump"
-        SKIPS: chunk_adaptive
+        SKIPS: chunk_adaptive histogram_test
```

Comment:
What happens on 32-bit systems?

Reply:
There is no overflow then, so `windows_bucket` returns a correct value, that is, does not generate an error.